File: CodeQuest2019 - Practice/Java/Prob04.java
import java.util.Scanner;

public class Prob04 {
    static final String key = "abcdefghijklmnopqrstuvwxyz";

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int ncases = Integer.valueOf(scanner.nextLine());
        for (int n = 0; n < ncases; n++) {
            String[] nums = scanner.nextLine().split(" ");
            int speed = Integer.valueOf(nums[0]);
            boolean bday = nums[1].equals("true");
            if (bday) {
                speed -= 5;
            }

            if (speed <= 60) {
                System.out.println("no ticket");
            } else if (speed >= 61 && speed <= 80) {
                System.out.println("small ticket");
            } else {
                System.out.println("big ticket");
            }
        }
    }
}

File: CodeQuest2019 - Practice/Elixir/Prob04.ex
main = fn () ->
  [a, b] = IO.gets("")
    |> String.trim
    |> String.split(" ")
   bday = b == "true"
   {dspeed, _} = Integer.parse(a)
   speed = cond do
     bday -> dspeed - 5
     true -> dspeed
   end

   cond do
     speed <= 60 ->
       IO.puts("no ticket")
     speed <= 80 ->
       IO.puts("small ticket")
     true ->
       IO.puts("big ticket")
   end
end




{n, "\n"} = IO.gets("")
  |> Integer.parse

for _ <- 1..n, do: main.()
